python/graphframes/tutorials/motif.py

```diff
@@ -0,0 +1,204 @@
+"""Demonstrate GraphFrames network motif finding capabilities. Code from the Network Motif Finding Tutorial."""
+
+#
+# Interactive Usage: pyspark --packages graphframes:graphframes:0.8.4-spark3.5-s_2.12
+#
+# Batch Usage: spark-submit --packages graphframes:graphframes:0.8.4-spark3.5-s_2.12 python/graphframes/tutorials/motif.py
+#
+
+import pyspark.sql.functions as F
+from pyspark import SparkContext
+from pyspark.sql import DataFrame, SparkSession
+
+from graphframes import GraphFrame
+
+# Initialize a SparkSession
+spark: SparkSession = SparkSession.builder.appName("Stack Overflow Motif Analysis").getOrCreate()
```

Comment:
What do you think about passing checkpoint dir during the session init and avoid at all any usage of SparkContext (in PySpark docs it is recommended to use SparkSession instead of SparkContext)?